### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by agbeltran**
## Description
Adding citation file; DOI will need to be added later.

## Testing instructions
- [ ] check text

## Agile board tracking
connect to #1297

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- family-names: "Glendenning"
+  given-names: "Sam"
+- family-names: "Kitenge"
+  given-names: "Joshua"
+- family-names: "Austin"
+  given-names: "Patrick"
+- family-names: "Richards"
+  given-names: "Matthew"
+- family-names: "Tim"
+  given-names: "Swinbank"
```
Comment (sam-glendenning) [inline CITATION.cff:19]: These need to be swapped

**Review by sam-glendenning — CHANGES_REQUESTED** [review]
Just happened to notice this PR so sorry if this is ahead of a formal review but thought I'd quickly add:

File: CITATION.cff (hunk 1 above)
Comment (agbeltran) [reply CITATION.cff:19]: @sam-glendenning thanks Sam!

**Review by sam-glendenning — APPROVED** [review]
@agbeltran this looks good but @louise-davies raised a concern in today's sprint review about whether or not this needed to go in the 1.0.0 release (which could cause difficulties since it's already been released). Apologies if I misremembered that, Louise, please feel free to jump in and clarify if so.

**Comment by agbeltran** [comment]
@sam-glendenning @louise-davies let's merge this in at the first opportunity

**Comment by sam-glendenning** [comment]
I'm happy with it, I believe Louise is too so will merge now

**Pull request merged**